include/modem/lte_lc.h

```diff
@@ -1345,18 +1345,96 @@ int lte_lc_normal(void);
  * Requested periodic TAU (RPTAU) and requested active time (RAT) are used when PSM mode is
  * subsequently enabled using lte_lc_psm_req().
  *
- * For reference see 3GPP 27.007 Ch. 7.38.
+ * For encoding of the variables, see nRF AT Commands Reference Guide, 3GPP 27.007 Ch. 7.38., and
+ * 3GPP 24.008 Ch. 10.5.7.4a and Ch. 10.5.7.3.
  *
- * @param[in] rptau Requested periodic TAU as a null-terminated string.
- *                  Set to @c NULL to use manufacturer-specific default value.
- * @param[in] rat Requested active time as a null-terminated string.
- *                Set to @c NULL to use manufacturer-specific default value.
+ * @param[in] rptau
+ * @parblock
+ *          Requested periodic TAU as a null-terminated 8 character long bit field string.
+ *          Set to @c NULL to use manufacturer-specific default value.
```

Comment:
I guess we should be more specific here what manufacturer means here? User might be wondering if it's LTE LC Kconfig options, or modem defaults.

Reply:
Yes, wouldn't hurt to be make it more precise. The "manufacturer-specific default" wording is also used in the AT command specification which implies that it refers to modem defaults. But, it is confusing because LTE LC also has it's own default values.

Reply:
I've changed "manufacturer-specific" default value to "modem default value"